pkg/controller/networkpolicy/store/addressgroup.go

```diff
@@ -157,7 +157,11 @@ func AddressGroupKeyFunc(obj interface{}) (string, error) {
 	return group.Name, nil
 }
 
+func addressGroupSelectorFunc(selectors *storage.Selectors, key string, obj interface{}) bool {
+	return filter(selectors, key, obj.(*types.AddressGroup).NodeNames)
```

Comment:
Question - not this PR but are the filter() funcs for all addressgroup, appliedtogroup, and networkpolicy the same? If so should we just define a single util func?

Reply:
filter is the util func, but yes, I think we can even save these addressGroupSelectorFunc funcs. Will update.